src/test/java/org/fcrepo/camel/tests/FcrepoLdpathIT.java

```diff
@@ -177,15 +177,13 @@ public void testLdpathService() throws Exception {
         final List<Map<String, List<String>>> data4 = MAPPER.readValue(response4, List.class);
 
         assertFalse(data4.isEmpty());
-        assertTrue(data4.get(0).containsKey("label"));
+        assertTrue(data4.get(0).containsKey("hasParent"));
         assertTrue(data4.get(0).containsKey("type"));
         assertTrue(data4.get(0).containsKey("id"));
-        assertTrue(data4.get(0).get("id").contains(url4));
-        assertTrue(data4.get(0).get("label").contains("Object 4"));
-        assertTrue(data4.get(0).get("type").contains("http://fedora.info/definitions/v4/repository#Container"));
+        assertTrue(data4.get(0).get("type").contains("http://fedora.info/definitions/v4/repository#Binary"));
         assertTrue(data4.get(0).get("type").contains("http://fedora.info/definitions/v4/repository#Resource"));
-        assertTrue(data4.get(0).get("type").contains("http://pcdm.org/models#Object"));
         assertTrue(data4.get(0).get("hasParent").contains(url2));
```

Comment:
Do you want to test for 'label' here as well, since rdfs:label is among the triples and in the default ldpath?  I don't view it as critical, since it's clear that the ldpath is working.  Just thinking that since it's there, why not test it?